code/game/jobs/job/z_all_jobs_vr.dm

```diff
@@ -47,3 +47,8 @@
 /datum/job/atmos
 	total_positions = 3
 	spawn_positions = 3
+	
+/datum/job/scientist
+	total_positions = 5
+	spawn_positions = 3
```

Comment:
Why positions defines?

Reply:
The file lacked a scientist entry. And seeing that every other entry in the file had position counts, I copied them over from scientist.dm.